modules/ipi-install-provisioning-the-bare-metal-node.adoc

```diff
@@ -10,47 +10,47 @@ Provisioning the bare metal node requires executing the following procedure from
 
 .Procedure
 
-. Ensure the `PROVISIONING STATUS` is `ready` before provisioning the bare metal node.
+. Ensure the `PROVISIONING STATUS` is `Available` before provisioning the bare metal node.
 +
-[source,bash]
+[source,terminal]
 ----
 $  oc -n openshift-machine-api get bmh openshift-worker-<num>
 ----
 +
 Where `<num>` is the worker node number.
 +
-[source,bash]
+[source,terminal]
 ----
 NAME                 STATUS   PROVISIONING STATUS   CONSUMER   BMC                 HARDWARE PROFILE   ONLINE   ERROR
-openshift-worker-<num>   OK       ready                            ipmi://<out-of-band-ip>   unknown            true
+openshift-worker-<num>   OK   Available                        ipmi://<out-of-band-ip>             unknown            true
```

Comment:
Please, use another example of output, it changed some time ago
NAME                   STATE         CONSUMER                                  ONLINE   ERROR   AGE
openshift-worker-<num>   available   ocp-edge-cluster-0-8rvrp-worker-0-g92k4   true             34h